app_recorder/launch/video_recorder.launch

```diff
@@ -3,10 +3,19 @@
   <arg name="video_path" />
   <arg name="video_title" />
   <arg name="video_topic_name" />
+  <arg name="use_compressed" default="false" />
+  <arg name="video_compressed_topic_name" />
```

Comment:
```suggestion
  <arg name="video_compressed_topic_name" default="" />
```

Reply:
we need to set compressed topic name even when you don't want to use compressed if we don't have the default value.

Reply:
Thank you for your advice.

I updated PR.